src/functions/uploadToRSIS/application/data-mapper/__tests__/helpers/cat-b/data-fields/fully-populated-data-fields.ts

```diff
@@ -0,0 +1,656 @@
+import { DataField } from '../../../../../../domain/mi-export-data';
+
+export function getFullyPopulatedFaultDataFields(): DataField[] {
```

Comment:
The three functions in this file have a fair bit of duplication of the same objects, perhaps we could loop through a single set of them passing through a value for how to set `val` in each case?

Reply:
Looking into this now, Ammar. Thanks for the suggestion.